src/ai/backend/storage/abc.py

```diff
@@ -77,6 +77,10 @@ async def get_capabilities(self) -> FrozenSet[str]:
     async def get_hwinfo(self) -> HardwareMetadata:
         pass
 
+    @abstractmethod
+    async def get_statistics(self) -> FSUsage:
```

Comment:
There is already `get_fs_usage()` abstract method. What's the difference here?;;
It just didn't have the URL handler, so let's just use/implement it for the new URL handler.